source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -186,6 +208,37 @@ content: |
            To use AWS KMS, you must use `pymongocrypt <https://pypi.org/project/pymongocrypt/>`__
            version 1.0 or later in your application's environment.
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+            var keyVaultClient = new MongoClient(connectionString);
+            var clientEncryptionOptions = new ClientEncryptionOptions(
+                keyVaultClient: keyVaultClient,
+                keyVaultNamespace: keyVaultNamespace,
+                kmsProviders: kmsProviders);
+
+            var clientEncryption = new ClientEncryption(clientEncryptionOptions);
+            var awsKeyARN = Environment.GetEnvironmentVariable("FLE_AWS_KEY_ARN"); // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
+            var awsKeyRegion = Environment.GetEnvironmentVariable("FLE_AWS_KEY_REGION");
```

Comment:
Can you check whether this issue also applies to C#: https://jira.mongodb.org/browse/DOCSP-12976

Reply:
This code works for me both when ran as a console application as well as within an Xunit test case.